chat-commands.js

```diff
@@ -692,7 +697,8 @@ const commands = {
 		let targetRoom = Rooms.createChatRoom(roomid, `[G] ${title}`, {
 			isPersonal: true,
 			isPrivate: privacy,
-			modjoin: '+',
+			modjoin: parent ? true : '+',
```

Comment:
Why would you set default to `true`?

Reply:
If they're a subroom they should not be invite only since the point is for them to be used in a room event. Do you want it to be null instead?

Reply:
`null` seems better than "surprise sync"